src/quo/components/text_combinations/style.cljs

```diff
@@ -16,3 +16,14 @@
    :letter-spacing 2
    :font-size      13
    :line-height    20.5})
+
+(defn textual-emoji
+  [size emoji-background-color]
+  {:border-radius    50
+   :border-width     0
+   :border-color     :transparent
+   :width            size
+   :height           size
+   :justify-content  :center
+   :align-items      :center
+   :background-color emoji-background-color})
```

Comment:
<img width="427" alt="Screenshot 2024-03-26 at 00 45 25" src="https://github.com/status-im/status-mobile/assets/22799766/66125df5-fca8-4e1f-91de-ef40e43a0dae">
If that is related to this, then this is 
`(colors/resolve-color customization-color theme 10)`